### ISSUE TITLE
Please rename ST2U settings menu entry  or integrate into Unified Settings window get rid of editor warnings in Unity 2019.1

### ISSUE THREAD

**Issue by SharlatanY** [Seanba/SuperTiled2Unity#66]
Currently, in Unity 2019.1.0f2, I constantly get the following warning in the editor when starting the player, recompiling the code, etc.:

> There are menu items registered under Edit/Project Settings: SuperTiled2Unity Settings
Consider using [SettingsProvider] attribute to register in the Unified Settings Window.
UnityEditor.EditorApplication:Internal_CallUpdateFunctions()

It would be very nice if you could integrate the unified settings window so we don't constantly get warnings in newer Unity versions. Obviously, it's not urgent :)

Thanks a lot!

**Comment by Seanba** [comment]
Thanks, @SharlatanY. I wasn't aware of unified settings. I'll try to find some time soon to fix this warning and see if the unified settings are easier/better to work with then the current way I'm handling ST2U settings (which is kind of a huge mess, honestly).

**Comment by SharlatanY** [comment]
@Seanba thanks for your quik feedback! And no problem! I think, Unity just doesn't like that you have an "Edit/Project Settings" menu entry and is totally okay with how you handle settings per se.

And btw., you don't like the way you handle the settings yourself? Just asking because I was thinking about taking a gander how you implemented the settings. I'm in need for some easily configurable settings for my main project as well and what I especially liked about your solution is the ability to e.g. be able to drag in prefabs. But if you don't recommend it, I'll be wary ;)

**Comment by Seanba** [comment]
Hi again, @SharlatanY.

> And btw., you don't like the way you handle the settings yourself?

I tried to handle settings with a scripted importer which in hindsight was a bad call. It should have been a scriptable object asset with a custom editor.

I've just pushed up version 1.4.0 of SuperTiled2Unity that rewrites how I store and manage settings and now that I know `SettingsProvider` is a thing (this is why it's helpful for people to write up such bugs!) I've put these settings in with other project-wide settings common in Unity. What I really like about this is the ST2U settings no longer take up the inspector window.

![image](https://user-images.githubusercontent.com/6980630/56867126-551e1300-69a7-11e9-98b3-6fb963eb706f.png)

Let me know if you run into any issues. One conceit is that you'll lose old settings with this change so you might want to take a screenshot or two ahead of time before you upgrade. Thanks.
